Pyblake2 package removal

ptagl opened the pull request:
Pyblake features are now integrated into the standard "hashlib" module, so `pyblake2` is not required to be installed/imported anymore.

Commit:
Pyblake2 package removal

Pyblake features are now integrated into the standard "hashlib" module.

JackPiri commented on contrib/ci-horizen/dockerfiles/amd64/linux/ubuntu_focal/Dockerfile:
```diff
@@ -16,7 +16,7 @@ RUN set -euxo pipefail \
     && apt-get -y --no-install-recommends install aria2 autoconf automake bsdmainutils build-essential ca-certificates \
        cmake curl dirmngr fakeroot git g++-multilib gnupg2 help2man libc6-dev libgomp1 libtool lintian m4 ncurses-dev \
        pigz pkg-config pv python3-dev python3-pip python3-setuptools python3-wheel time unzip wget zlib1g-dev \
-    && pip3 install pycryptodome b2 pyblake2 pyzmq websocket-client pyzmq \
```
Just seen by chance, can we get rid of that `pyzmq` being written a second time?

ptagl replied:
Nice catch, removed, thanks!

JackPiri reviewed (approved):
The PR is fine to me.
Thanks

Commit:
Removed pyblake2 module from Joinsplit test script

One of the Joinsplit unit tests has attached a Python script to generate test vectors. As it was using the pyblake2 module, the script has been updated to use hashlib instead.

The script is also now compatible with Python 3.

Commit:
[Docker] Removed redundant installation of pyzmq module

drgora reviewed (approved):
Looks good to me

a-petrini reviewed (approved):
Looks good, thanks for his PR!

titusen reviewed (approved):
LGTM, thanks.
